flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/TableEnvironment.scala

```diff
@@ -587,6 +587,18 @@ abstract class TableEnvironment(val config: TableConfig) {
     rootSchema.getTableNames.asScala.toArray
   }
 
+  def listViews(): Array[String] = {
```

Comment:
I think we can't add this function in `TableEnvironment`. Reasons are:
- `table.isInstanceOf[RelTable]` is not necessary a View. `RelTable` can be registered through `registerTable()`. 
- `TableEnvironment` currently doesn't support view. It is only supported in SQL-Client now.

As a choice, you can list views directly from the `session` passed to the `listViews()`. Also pay attention to the order.

Reply:
Do you mean that I use the original implementation directly, just move it to the `Executor`? I used to get from the session, I misunderstood what you mean, I thought it was to refer to the implementation of the `listTable` method.